### ISSUE TITLE
Make Icon Panel of classic block expanded as default

### ISSUE THREAD

**Issue by La-Geek** [WordPress/gutenberg#10508]
When you open the Classic Editor block for the first time, the icon field is completely collapsed. 
If you open the classic editor for the second time and so on, "only" the second icon bar is collapsed.

This is annoying. Should be expanded by default when opened

These clicks are unnecessary, but must be executed every time.

Also: WordPress beginners will have a hard time with it.

![icon_panel](https://user-images.githubusercontent.com/1585416/46803065-7860d800-cd5f-11e8-9f4b-6d7dd7cf41e2.gif)

**Comment by Soean** [comment]
I think it makes sense to hide the toolbar, if the block is not selected. If you click inside the block, the toolbar shows up. 

If you insert a classic block, the block should be selected by default.

If you open the second row in the classic editor it stays open in all posts, maybe we should add this behaviour to the classic block?

**Comment by La-Geek** [comment]
> I think it makes sense to hide the toolbar, if the block is not selected. If you click inside the block, the toolbar shows up.
> If you insert a classic block, the block should be selected by default.

Yes, make sense -> "the block should be selected by default" and be expanded

> If you open the second row in the classic editor it stays open in all posts, maybe we should add this behaviour to the classic block?

Imho both rows should be expanded by default, when the classic block is selected.

**Comment by Soean** [comment]
> Imho both rows should be expanded by default, when the classic block is selected.

The important icons are in the first row. If you want more, you can toggle the second row. Thats the same like in 4.9. The difference is that the classic editor saves the toggle status, the classic block does't do this yet.

**Comment by La-Geek** [comment]
Got it!

> If you open the second row in the classic editor it stays open in all posts, maybe we should add this behaviour to the classic block?

Yes. Please :)

**Comment by ghost** [comment]
It seems like this is no longer the case. If you think there's still an issue, feel free to reopen this.

https://user-images.githubusercontent.com/118618474/222987424-f1941d92-1bd1-4e86-bda9-29558a7239d4.mov
